src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -38,12 +38,15 @@
     <!-- Update nuget.config property to point to modified file that will be created/updated during build. -->
     <NuGetConfigFile Condition="'$(OriginalNuGetConfigFile)' != ''">$(BaseIntermediateOutputPath)$([System.IO.Path]::GetFileName('$(OriginalNuGetConfigFile)'))</NuGetConfigFile>
 
+    <RepoAssetManifestsDir>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)', 'AssetManifests'))</RepoAssetManifestsDir>
+
     <SourceBuiltSdksDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'source-built-sdks'))</SourceBuiltSdksDir>
+
+    <RepoPackagesTempDir>$(BaseIntermediateOutputPath)Packages/</RepoPackagesTempDir>
```

Comment:
This is now unused:

```suggestion
```

Reply:
Will fix - already have other pending changes so it's easier to push this as well.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9c2058b933a4dac3610accdcceff226d11565198